grafana/grafana.spec

```diff
@@ -0,0 +1,309 @@
+Name:             grafana
+Version:          6.2.2
```

Comment:
there is a 6.2.4 version upstream. Do we want to be stick on 6.2.2?

Reply:
I think 6.2.2 is what's being shipped with OpenShift 4.1, other than that I'm ok either way.